app/assets/stylesheets/custom.scss

```diff
@@ -61,7 +61,7 @@ table a, a:hover{
 }
 
 .welcome-message {
-  color: black;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.